modules/data.atmosphere/R/download.raw.met.module.R

```diff
@@ -0,0 +1,29 @@
+.download.raw.met.module <- function(dir, met, register, machine, start_date, end_date, con, 
```

Comment:
Only because I envisioned that the generic super-module would be the interface and details would be hidden away. But now that you mention it I could see someone just wanting to use this function rather than figure out where the desired dataset is hidden on the web...